Pearson-Advance/openedx-themes
# PAE-156 - Add the Regis theme.

## Description:

This PR adds the base theme for the Regis site.

## Screenshots:

![image](https://user-images.githubusercontent.com/17520199/176222888-6306ac38-7a0c-4bbe-9f43-9902839d1505.png)
![image](https://user-images.githubusercontent.com/17520199/176223937-35b493a3-4135-47f0-adbd-7d2f9d368a68.png)
![image](https://user-images.githubusercontent.com/17520199/176225789-31da0e1e-6438-4db1-8c39-6c2ee7f2b031.png)

## Note:

- This theme is based on the Pathways theme.
- Footer is expected to be changed.

## Testing:

- Check out to this branch.
- Enable Comprehensive theming: https://edx.readthedocs.io/projects/edx-installing-configuring-and-running/en/latest/configuration/changing_appearance/theming/enable_themes.html
- Compile assets, `paver update_assets --themes=pearson-regis-theme`
- Assign 'pearson-regis-theme' to a new site.

## Reviewers:
